hw/dv/dpi/spidpi/spidpi.h

```diff
@@ -5,10 +5,16 @@
 #ifndef OPENTITAN_HW_DV_DPI_SPIDPI_SPIDPI_H_
 #define OPENTITAN_HW_DV_DPI_SPIDPI_SPIDPI_H_
 
+#ifdef __linux__
+#include <linux/limits.h>
+#endif
```

Comment:
Hmm, what does `spidpi.h` include (any) `limits.h` file for? Do we definitely need both of these in the header (rather than e.g. `spidpi.c`)?

Reply:
Hm... probably the use of PATH_MAX in the definition of `struct spidpi_ctx` below. Do we actually need to save the path to the file there, though?

Reply:
Actually... on top of that, spidpi_ctx can go entirely in spidpi.c. It seems like it was meant to be an opaque type--There are no users that access the members other than spidpi.c.

Reply:
Yes, PATH_MAX needs linux/limits.h, probably in Centos. And it seems clear spidpi.h can be much leaner. Same for uartdpi.h. I will leave usbdpi alone :)